style_editor.html

```diff
@@ -9629,7 +9645,11 @@
 }
 
 function AddTabContent(tab, data) {
-  FIND(tab + "_tabcontent").innerHTML += data;
+  var tabContentElement = FIND(tab + "_tabcontent");
+  tabContentElement.innerHTML += data;
+  if (tab === 'rgb') {
+    tabContentElement.classList.add('rgb-tabcontent');
```

Comment:
Why not just add this in the HTML?

Reply:
unsure

Reply:
unsure about what?

Reply:
That, or you could target your style to the ID.

Reply:
OH I looked right past the fact we're already adding class=tabcontent already right there in AddTab.
But now I'm thinking why target only rgb "Color" tab, when we could just do it for all tabs, leaving the option to style them later if desired. (like the buttons could be styled differently if the tab's buttons have the long strings etc...)
Change applied.